Assets/MRTK/SDK/Features/Input/Handlers/ObjectManipulator.cs

```diff
@@ -707,9 +861,30 @@ private void ApplyTargetTransform(MixedRealityTransform targetTransform)
         {
             if (rigidBody == null)
             {
-                HostTransform.position = smoothingActive ? Smoothing.SmoothTo(HostTransform.position, targetTransform.Position, moveLerpTime, Time.deltaTime) : targetTransform.Position;
-                HostTransform.rotation = smoothingActive ? Smoothing.SmoothTo(HostTransform.rotation, targetTransform.Rotation, rotateLerpTime, Time.deltaTime) : targetTransform.Rotation;
-                HostTransform.localScale = smoothingActive ? Smoothing.SmoothTo(HostTransform.localScale, targetTransform.Scale, scaleLerpTime, Time.deltaTime) : targetTransform.Scale;
+                if (elasticTypes.HasFlag(TransformFlags.Move))
```

Comment:
Smoothing is not applied at all if elastic is enabled. This is because elastic essentially serves the same purpose as smoothing, and applying both may result in unwanted sluggishness when interacting. I am open to discussion and input on this decision!